src/components/ladder/RankingsGrid.vue

```diff
@@ -357,15 +357,19 @@ export default class RankingsGrid extends Vue {
           behavior: "smooth",
         });
       }
-    }, 200);
+    }, 500);
```

Comment:
@Cepheid-UK 

The scroll was occurring 0.2 seconds after the player was selected in the search box. This would fail sometimes when the rankings for the selected player's league hadn't finished refreshing. I increased the scroll delay to 0.5 seconds to give time for the rankings to finish refreshing from the API.

Reply:
It does seem like this will be a good change, but it's still not fixed the bug with not scrolling correctly :(

Reply:
Let me troubleshoot this some more.

Did this one work for you? **"2. When a user clicks on the division icon on a player's profile, the rankings will open and scroll to the player"**

Reply:
this is still ongoing, right? I marked it as a draft, just to not merge this accidentally.

Reply:
@modmoto  yes